spec.html

```diff
@@ -26273,7 +26273,7 @@ <h1>Function.prototype.bind ( _thisArg_, ..._args_ )</h1>
             1. If Type(_targetLen_) is not Number, let _L_ be 0.
             1. Else,
               1. Set _targetLen_ to ! ToInteger(_targetLen_).
-              1. Let _L_ be the larger of 0 and the result of _targetLen_ minus the number of elements of _args_.
+              1. Let _L_ be the larger of *+0* and the result of _targetLen_ minus the number of elements of _args_.
```

Comment:
I don't think this line needs to change. There's no need to distinguish +0 from 0 here.

Reply:
I can certainly revert it; it's implicitly `+0` already, but #1086 intends to explicitly sign as many zeroes as possible.

Reply:
There's this sentence already in the spec:

> The Number value 0, alternatively written 0𝔽, is defined as the double-precision floating point positive zero value. In certain contexts, it may also be written as +0 for clarity.

When I see +0, I'm personally primed to think this is a place where I also need to care about -0. If it doesn't matter, it'd be better editorially to not distinguish IMO.

Reply:
It matters in the sense that the length of a bound function should never be -0.